<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Home">
     <widget class="QToolButton" name="aboutButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>About</string>
      </property>
     </widget>
     <widget class="QToolButton" name="videoConvButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Video Converter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Silahkan pilih fitur yang anda inginkan</string>
      </property>
     </widget>
     <widget class="QToolButton" name="imageConvButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Image Converter</string>
      </property>
     </widget>
     <widget class="QToolButton" name="audioConvButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Audio Converter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="InputConverter">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>File :</string>
      </property>
     </widget>
     <widget class="QToolButton" name="inputClear">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pilih tipe format :</string>
      </property>
     </widget>
     <widget class="QLabel" name="inputLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pilih file dengan format .xxx untuk di konversi</string>
      </property>
     </widget>
     <widget class="QLabel" name="inputConverterTypeLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;&lt;NAME CONVERTER&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="inputRadioButton1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="inputRadioButton2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="inputRadioButton3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QToolButton" name="inputNextButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QToolButton" name="inputBackButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QToolButton" name="inputBrowseFile">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse..</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputPath">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>70</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="OutputConverter">
     <widget class="QToolButton" name="outputBrowseFile">
      <property name="geometry">
       <rect>
        <x>356</x>
        <y>90</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse..</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Folder :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pilih folder tempat menyimpan file hasil konversi : </string>
      </property>
     </widget>
     <widget class="QToolButton" name="outputStartButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QToolButton" name="outputBackButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QToolButton" name="outputClear">
      <property name="geometry">
       <rect>
        <x>296</x>
        <y>90</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="outputPath">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>90</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="outputFilename">
      <property name="geometry">
       <rect>
        <x>76</x>
        <y>130</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>130</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Filename :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="LoadingPage">
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Harap tunggu . . .</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sedang mengonversi file . . .</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ResultPage">
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>File telah selesai di konversi !!!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Silahkan cek folder yang dimaksud :</string>
      </property>
     </widget>
     <widget class="QLabel" name="resultPath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;PATH&gt;&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="resultBackButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back To Home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="About">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Program ini dibuat oleh :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Faishal Azka Jellyanto</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Raga Krilido</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>R. M Iskandar Z</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>M. Ardhinata J (Kontributor)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>5112100061</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>5112100064</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>5112100101</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>5112100094</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tugas Jaringan Multimedia A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Teknik Informatika ITS 2015</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Copyright © All Rights Reserved 2015</string>
      </property>
     </widget>
     <widget class="QToolButton" name="aboutBackButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>443</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
